<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointerMotionForm</class>
 <widget class="QWidget" name="PointerMotionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pointer Motion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string comment="Mouse pointer motion parameters">Speed</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string comment="Minimum pointer movement speed">Minimum:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_MinSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CustomSlider" name="kcfg_MinSpeed">
        <property name="toolTip">
         <string>Minimum pointer speed</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string comment="Maximum pointer movement speed">Maximum:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_MaxSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CustomSlider" name="kcfg_MaxSpeed">
        <property name="toolTip">
         <string>Maximum pointer speed</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string comment="Pointer acceleration">Acceleration:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_AccelFactor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CustomSlider" name="kcfg_AccelFactor">
        <property name="toolTip">
         <string>Acceleration factor for normal pointer movements</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string comment="Pointer motion">Pressure-Dependent Motion</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string comment="Pressure-dependent pointer motion">Minimum pressure:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_PressureMotionMinZ</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string comment="Pressure-dependent pointer motion">Minimum factor:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_PressureMotionMinFactor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string comment="Pressure-dependent pointer motion">Maximum pressure:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_PressureMotionMaxZ</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string comment="Pressure-dependent pointer motion">Maximum factor:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_PressureMotionMaxFactor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="kcfg_PressureMotionMinZ">
        <property name="toolTip">
         <string>Finger pressure at which minimum pressure motion factor is applied</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="kcfg_PressureMotionMaxZ">
        <property name="toolTip">
         <string>Finger pressure at which maximum pressure motion factor is applied</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_PressureMotionMinFactor">
        <property name="toolTip">
         <string>Lowest setting for pressure motion factor</string>
        </property>
        <property name="suffix">
         <string notr="true" comment="Speed multiplier suffix"> x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="kcfg_PressureMotionMaxFactor">
        <property name="toolTip">
         <string>Greatest setting for pressure motion factor</string>
        </property>
        <property name="suffix">
         <string notr="true" comment="Speed multiplier suffix"> x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string comment="Pointer motion">Noise Cancellation</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string comment="Noise cancellation">Vertical:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_VertHysteresis</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string comment="Noise cancellation">Horizontal:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_HorizHysteresis</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="kcfg_VertHysteresis">
        <property name="toolTip">
         <string>The minimum vertical hardware distance required to generate motion events</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_HorizHysteresis">
        <property name="toolTip">
         <string>The minimum horizontal hardware distance required to generate motion events</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="suffix">
         <string> units</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QSlider</extends>
   <header>kcm/xlib/customslider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_MinSpeed</tabstop>
  <tabstop>kcfg_MaxSpeed</tabstop>
  <tabstop>kcfg_AccelFactor</tabstop>
  <tabstop>kcfg_PressureMotionMinZ</tabstop>
  <tabstop>kcfg_PressureMotionMinFactor</tabstop>
  <tabstop>kcfg_PressureMotionMaxZ</tabstop>
  <tabstop>kcfg_PressureMotionMaxFactor</tabstop>
  <tabstop>kcfg_VertHysteresis</tabstop>
  <tabstop>kcfg_HorizHysteresis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
